chalice/pipeline.py

```diff
@@ -31,10 +49,21 @@ def __init__(self):
         # type: () -> None
         pass
 
-    def create_template(self, app_name):
-        # type: (str) -> Dict[str, Any]
+    def _codebuild_image(self, lambda_python_version):
+        # type: (str) -> str
+        try:
+            image = self._CODEBUILD_IMAGE[lambda_python_version]
+            return image
+        except KeyError as e:
+            raise InvalidCodeBuildPythonVersion(str(e))
+
+    def create_template(self, app_name, python_lambda_version):
```

Comment:
This is fine for now, but one of the things that I've noticed is that these arguments tend to grow over time as more things are supported.  They eventually just end up taking `config`.  Just something to keep in mind for future cases.